modules/Bio/Otter/Lace/Defaults.pm

```diff
@@ -687,7 +687,7 @@ __DATA__
 
 [client]
 url=http://localhost:8004/cgi-bin/otter
-write_access=0
+write_access=1
```

Comment:
Can't we avoid changing default access to write? It looks like we skip our security checks.

Reply:
At the moment, no. We will have to include this. The security checks are placed elsewhere.